packages/pytest-simcore/requirements/tests_base.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Essential testing libraries and some additional plugins
+#
+pytest
+
+# enhances tests logs
+pytest-icdiff
+pytest-instafail
+pytest-sugar
+
+# enhances test runs
+pytest-xdist
+pytest-runner # TODO: check if REQUIRED ANYMORE ???
+
+# adds functionality
+pytest-lazy-fixture
+
+
+# support testing features built on frameworks
+# pytest-docker
```

Comment:
you could add pytest-black and maybe pytest-mypy (but I'm not sure about this one)

Reply:
will look into those, thx